Sources/SwiftDocC/Model/Rendering/RenderSectionTranslator/DeclarationsSectionTranslator.swift

```diff
@@ -11,8 +11,57 @@
 import Foundation
 import SymbolKit
 
+typealias OverloadDeclaration = (
+    declaration: [SymbolGraph.Symbol.DeclarationFragments.Fragment],
+    reference: ResolvedTopicReference
+)
+
 /// Translates a symbol's declaration into a render node's Declarations section.
 struct DeclarationsSectionTranslator: RenderSectionTranslator {
+    /// A mapping from a symbol reference to the "common fragments" in its overload group.
+    static let commonFragmentsMap: Synchronized<[ResolvedTopicReference: [SymbolGraph.Symbol.DeclarationFragments.Fragment]]> = .init([:])
```

Comment:
Why define this as static? It will work fine, I think, but would fail someday if we ever compile more than one documentation project in the same DocC process, because we would have values left over from previous runs. As written, we assume the map will only ever be used once. Might be safer to just make this a normal attribute of `DeclarationsSectionTranslator`.

Reply:
[`DeclarationsSectionTranslator` instances are not shared between different symbols' renderings.](https://github.com/swiftlang/swift-docc/blob/2e3e73a80a6e0d460e9d6d323020757a1e0dfa12/Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift#L1356) If i made this an instance property, the saved results would not be available the next time around.

Reply:
I see, ok makes sense.